src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -800,6 +835,18 @@ def start_analyze_pointsource(request, format=None):
     wkaoi = request.query_params.get('wkaoi', None)
     area_of_interest = load_area_of_interest(request.data, wkaoi)
 
+    if not validate_analyze_aoi_size_below_max_area(area_of_interest):
```

Comment:
It occurred to me that we could just add a single `validate_aoi` function to group all these together in the `validations.py` file then run it there.

Reply:
Done in 45de649